lib/services/ngsi/ngsiService.js

```diff
@@ -318,6 +318,22 @@ function sendQueryValue(entityName, attributes, typeInformation, token, callback
             }));
 }
 
+/**
+ * Update the trust for a device or a deviceGroup depending on the source of the trust token.
+ * Currently is a placeholder, but it is needed in case of Auth services which tokens have
+ * expiration.
+ *
+ * @param {Object} deviceGroup        The deviceGroup where the trust token was stored. Null if stored in actual device.
+ * @param {Object} deviceInformation  The device where the trust token was stored. Null if stored in deviceGroup.
+ * @param {String} trust              The current trust token.
+ * @param {Object} response           The response where the new token is stored.
+ * @param {Function} callback         The callback function.
+ */
+
+function updateTrust(deviceGroup, deviceInformation, trust, response, callback) {
+    callback(null, response);
```

Comment:
@fgalan this is a place holder, in case we have an authentication mechanism that require update of `trust`. can be removed if you think it is not worth to keep it at the time being.

Reply:
I think is a good idea.

NTC